docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -757,6 +757,11 @@ To collect statistics for a table, execute the following statement::
 
   ANALYZE table_schema.table_name;
 
+Because Delta Lake connector can also update stats on writes they can drift away from
```

Comment:
Drifting from proper values are because we don't update statistics in some situations not the opposite.